### PR TITLE
New PR template

### PR DESCRIPTION
**Pull request by Ignition**
New PR template for working with the merge queue (title + number + description)

### CODE REVIEW

**Comment by Ignition** [comment]
### Tracking
- [ ] **[Link to Epic/Issue]**


### Standard development
- [ ] Update unit/E2E tests
- [ ] Compare the [benchmarking results](https://bench-graph.memgraph.com/) between the master branch this branch


### CI Testing Labels
- [ ] Select the appropriate CI test labels _(CI -build=**build-name** -test=**test-suite**)_


### Documentation checklist
- [ ] Add the documentation label
- [ ] Add the bug / feature label
- [ ] Add the milestone for which this feature is intended
    - If not known, set for a later milestone
- [ ] Write a release note, including added/changed clauses
    - **[ Release note text ]**
- [ ] **[ Documentation PR link memgraph/documentation#XXXX ]**
    - [ ] Is back linked to this development PR
- [ ] **[ Tag someone from docs team ]**

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -1,32 +1,30 @@
-### Description
+Git commit description, explain the changes you made here
 
-Please briefly explain the changes you made here.
+---
+__*Leave above in PR description, copy the below into a comment*__
+___
```
Comment (andrejtonev) [inline .github/pull_request_template.md:5]: Someone will 100% leave the `Git commit description, explain the changes you made here` part in.

Comment (Ignition) [reply .github/pull_request_template.md:5]: Perfect is the enemy of good enough

**Review by andrejtonev — APPROVED** [review]

**Review by MarkoBarisic — APPROVED** [review]
LGTM

**Pull request merged**